.devcontainer/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM gcr.io/envoy-ci/envoy-build:04f06115b6ee7cfea74930353fb47a41149cbec3
+
+ARG USERNAME=vscode
+ARG USER_UID=501
+ARG USER_GID=$USER_UID
+
+ENV BUILD_DIR=/build
```

Comment:
For whatever reason, when working with this on Linux, I needed to chmod this to be a+w (or alternatively put in vscode owner/group)

Reply:
added in setup.sh to do chown